pkg/skaffold/initializer/prompt/prompt.go

```diff
@@ -47,13 +47,22 @@ func buildConfig(image string, choices []string) (string, error) {
 	return selectedBuildConfig, nil
 }
 
-func WriteSkaffoldConfig(out io.Writer, pipeline []byte, filePath string) (bool, error) {
+func WriteSkaffoldConfig(out io.Writer, pipeline []byte, generatedManifests map[string][]byte, filePath string) (bool, error) {
 	fmt.Fprintln(out, string(pipeline))
 
+	for f, m := range generatedManifests {
```

Comment:
Are these generated manifests going to be written in `skaffold.yaml` file?

Reply:
no, they're paired up with filepaths when they're generated, and then written to those paths. this prompting function is just to retrieve confirmation from the user before we do the actual file writing